_posts/2016/2016-07-07-elastic-meetup.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Elastic Meetup at ePages"
+date: "2016-07-07 07:16:17"
+categories: events
+authors: ["Christian K.", "Jens"]
+---
+
+On the 30th of June the third [jenadevs](http://www.meetup.com/de-DE/jenadevs/) meetup took place at the ePages office in Jena. A video conference with our office in Hamburg was also established to give more people the option to attend even remotely.
+
+Approximately 30 participants were interested in the topic and showed up at 6 pm.
+After a short general introduction the presentations started:
+
+## Elastic Stack 5.0
+
+[Pere Urbón-Bayes](http://www.purbon.com/) - [elastic](https://www.elastic.co)
+
+Pere walked through the new features of the Elastic 5 stack and explained for every component what's new and what changed.
+Further information on this topic: [elastic v5](https://www.elastic.co/de/v5).
+
+## Visualisation and analysis of Open Data
+
+[Achim Friedland](https://twitter.com/ahzf) - [OK Lab Jena](http://codefor.de/jena/)
+
+Achim presented the [CODE for Germany](http://codefor.de/) project and how it uses open data to visualise and solve problems.
+He showed some examples of visualised Open Data.
+
+## Elastic Stack @epages
+
+[Christian Köhler](https://twitter.com/epagesdevs) - [epages](http://www.epages.com/)
+
+Christian presented the current Elastic Stack at ePages.
+He explained the approach and the lessons learned to build the centralised logging framework.
+
+## Regression Test Analysis with Elastic Search in an Continuous Delivery Pipeline
+
+[Benjamin Nothdurft](https://twitter.com/dataduke) and [Bastian Klein](https://twitter.com/Dastianoro) - [epages](http://www.epages.com/)
```

Comment:
[Benjamin Nothdurft](https://twitter.com/dataduke) and [Bastian Klein](https://twitter.com/Dastianoro) presented a short version of their implementation of the Elasticsearch, Logstash and Docker integration for aggregation and collection of integration test results from various systems in the delivery pipeline ([presentation](https://speakerdeck.com/dataduke/automated-test-evaluation-short-version)).